src/ast.ml

```diff
@@ -107,6 +108,7 @@ let rec string_of_expr = function
   | BoolLit b -> if b then "true" else "false"
   | CharLit c -> "'" ^ Char.escaped c ^ "'"
   | StringLit s -> "\"" ^ s ^ "\""
+  | ListLit el -> "[" ^ String.concat ", " (List.map string_of_expr el) ^ "\""
```

Comment:
Should the closing symbol be a `]` instead of `"`?

Reply:
yea lol